InstaClone/Models/UserModel.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  UserModel.swift
+//  InstaClone
+//
+//  Created by MJ Albdour on 2/23/22.
+//
+
+import Foundation
+
+struct User {
+    let id: Int
+    let name: String
+    let gender: String
+    let dateOfBirth: Date
+    let active: Bool
+    let locale: String
+    let mobileNumber: String
+    let username: String
+    let email: String
+    let bio: String
+    let verified: String
+    let accountType: AccountType
+    let country: String
+
+    init(id: Int, name: String, gender: String,
```

Comment:
Try to reduce the size of the constructor in the future, perhaps use the builder pattern.